src/mos.c

```diff
@@ -162,7 +170,7 @@ BYTE mos_getkey() {
 //
 UINT24 mos_input(char * buffer, int bufferLength) {
 	INT24 retval;
-	putch(MOS_prompt);
+	printf("%s%c", cwd, MOS_prompt);

```

Comment:
the coloured directory name present in rainbow MOS isn't here - that's because it was blindly changing colours, and thus wouldn't respect the user's current FG text colour choice, replacing it on every new line